src/develop/blend_gui.c

```diff
@@ -844,6 +866,105 @@ static gboolean _blendop_blendif_draw(GtkWidget *widget, cairo_t *cr, dt_iop_mod
   return FALSE;
 }
 
+// magic mode: if mouse curser enters a gradient slider with shift and/or control pressed we
```

Comment:
That is so in dt-style :)
I'll be surprised if we can not deliver proper keypress here, after some modifications.